### PR TITLE
created init scripts

### PR DESCRIPTION
**Pull request by J-avery32**
I finished creating the initialization scripts. The process for how to initialize the system should be described in INIT.MD

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ services:
     - 80:80
     - 443:443
     - 9200:9200
+    - 9300:9300
```
Comment (dylanmcreynolds) [inline docker-compose.yml]: I really don't think we want to open either 9200 or 9300. They should be hidden behind nginx under /search. Also, I think 9300 is only necessary if you have elastic clusters.

File: INIT.MD [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Make sure to create all of the folders that will be used by the volumes first. If you don't it creates weird permissions issues. Also create a file called `monstache.config.toml` in the monstache volume directory. It may be necessary to set `elasticsearch-healthcheck-timeout` and `elasticsearch-healthcheck-timeout-startup` to larger values in this config file depending on how slow your computer is.
+
+On Linux hosts run  `sysctl -w vm.max_map_count=262144`  and then `sysctl --system`
+
+See: https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-prod-cluster-composefile
+
+Change your working directory to the top level directory
+
+These commands initialize the replica set for mongo:
```
Comment (dylanmcreynolds) [inline INIT.MD:21]: I was hoping for a script that performed all of these for the user. But this is a good start.

File: INIT.MD [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Make sure to create all of the folders that will be used by the volumes first. Here, we are using /splash/data/mongo and /splash/data/elastic and /splash/data/monstache
+
+Create a .env file and add the following (replacing with your folder names)
+
+MONGO_DB_LOC=/splash/data/mongo
+ELASTIC_DATA_LOC=/splash/data/elastic
+MONSTACHE_DATA_LOC=/splash/data/monstache
+
+
+Now create a file called `monstache.config.toml` in the monstache volume directory. It may be necessary to set `elasticsearch-healthcheck-timeout` and `elasticsearch-healthcheck-timeout-startup` to larger values in this config file depending on how slow your computer is.
```
Comment (dylanmcreynolds) [inline INIT.MD]: Please include a sample of this toml configuration in the documentation What is a suggestion for a larger value? Is this whole toml thing an optional step? If so, please describe that and describe the circumstance where you'd need it?

Comment (dylanmcreynolds) [reply INIT.MD]: Also, does this toml file go in /splash/data/monstache or /splash/data/monstache/data?

File: INIT.MD (hunk 2 above)
Comment (J-avery32) [reply INIT.MD:21]: The problem here is that for some reason this stuff won't work within a script. There's an issue with it attempting to do stuff before the other service is ready.

Comment (J-avery32) [reply INIT.MD:21]: As in it will start up mongo, and then attempt to execute code on the mongo instance before mongo is fully running.

**Pull request closed without merging**